charts/pega/README.md

```diff
@@ -656,6 +659,8 @@ If you are planning to use Cassandra (usually as a part of Pega Customer Decisio
 
 To use an existing Cassandra deployment, set `cassandra.enabled` to `false` and configure the `dds` section to reference your deployment.
 
+If you configured a secret in an external secrets operator, enter the secret name in external_secret_name parameter . For details, see [this section.](#optional-support-for-providing-credentialscertificates-using-external-secrets-operator)
```

Comment:
very minor>> fix space before period and add ticks to parameter name like you did in line 123>
If you configured a secret in an external secrets operator, enter the secret name in `external_secret_name` parameter#.

Reply:
Fixed